lib/db/DB.ts

```diff
@@ -57,8 +57,9 @@ class DB {
 
     if (newDb && initDb) {
       // populate new databases with default data
+      const firstStagePromises: Bluebird<any>[] = [];
```

Comment:
Why not just `promises`?

Reply:
Oops, that's kind of a leftover from when I was using a different approach with two arrays of promises, will correct it.